api/lib/subscription/slashing.js

```diff
@@ -27,10 +27,15 @@ class SlashingMonitor {
             networkId: this.networkId,
             operatorAddress: address,
             reason: tendermintResponse.events['slash.reason'][index],
-            amount: this.api.reducers.coinReducer({
-              amount: tendermintResponse.events['slash.power'][index],
-              denom: this.network.stakingDenom
-            }),
+            amount: this.api.reducers.coinReducer(
+              {
+                amount: tendermintResponse.events['slash.power'][index],
+                denom: this.network.stakingDenom
+              },
+              {
+                chainToViewConversionFactor: 0.000001 // this chainToViewConversionFactor should be accurate for stakingDenoms
```

Comment:
why is this a fixed value here? wouldn't that cause the same issues in the future?

Reply:
true. I was only thinking of Tendermint networks. It shouldn't be fixed, you are right

Reply:
Done